rotkehlchen/exchanges/coinbase.py

```diff
@@ -601,11 +601,21 @@ def _deserialize_asset_movement(self, raw_data: Dict[str, Any]) -> Optional[Asse
             # movement_category: Union[Literal['deposit'], Literal['withdrawal']]
             if 'type' in raw_data:
                 # Then this should be a "send" which is the way Coinbase uses to send
-                # crypto outside of the exchange
+                # crypto outside of the exchange, or from one user to another.
                 # https://developers.coinbase.com/api/v2?python#transaction-resource
-                msg = 'Non "send" type found in coinbase deposit/withdrawal processing'
-                assert raw_data['type'] == 'send', msg
-                movement_category = AssetMovementCategory.WITHDRAWAL
+                raw_type = raw_data.get('type', '')
+                if raw_type == 'send':
+                    movement_category = AssetMovementCategory.WITHDRAWAL
+                elif raw_type == 'inflation_reward':
+                    movement_category = AssetMovementCategory.DEPOSIT
```

Comment:
I believe these ASSET movements need to be changed to INCOME ledger actions to appear as taxable activity on the P&L.  So I have to refactor this based on the identification added here, and on L618.